src/components/designer/AutoMineButton.tsx

```diff
@@ -0,0 +1,142 @@
+import { FieldTimeOutlined } from '@ant-design/icons';
+import styled from '@emotion/styled';
+import { Button, Dropdown, Tooltip, MenuProps } from 'antd';
+import { ItemType } from 'antd/lib/menu/hooks/useItems';
+import { usePrefixedTranslation } from 'hooks';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useStoreActions, useStoreState } from 'store';
+import { AutoMineMode, Network } from 'types';
+
+const barCssTransition = 'width 400ms ease-in-out';
```

Comment:
We don't need to make this a variable since it's only used one time on line 17. I think this was leftover from prior commits.